notes/git-guidelines-commit-and-push.md

```diff
@@ -0,0 +1,74 @@
+## GUIDELINES TO GIT COMMIT AND PUSH
+
+Committing and pushing changes in Git involve several best practices to ensure a clean and organized version history. Here are some guidelines and best practices:
+
+### COMMITTING
+
+1. always check if you are in the right branch
```

Comment:
Not whitespaces between 1.Always - Always capital latter not lowercase
check readme here: https://dillinger.io/